src/enpass_fmt_plug.c

```diff
@@ -66,8 +66,7 @@ john_register_one(&fmt_enpass);
+static struct fmt_tests enpass_tests[] = {				  {"$enpass$0$100000$73a0f3720c04e61c26a4062e4066a31143a8dd07540518981106cf7b9ec8c9b62d91803f0319c916056727309f4ac9adc9177a3c5e9c6873635e2512af9481443cc85362b67af2a14e32bdd063bd815276669ec17d4d23e1cd8859024eaeac1de36138fabfd001335be3b98b4404259c40a5797bca8763fd72a0cb5ba8ad06ab94677eaa9b4c7d2753b52c2caba82207a0e2b0213ecad22c4249ce6c708d0163d988d0cb2569b975b07ad7d7b773967f05a8a9392526e0b465f4e60c3a8021b346390f654384b58dda041b90bd1e6e3843ad62073e02bab8eba7d8ef9e1dac338890666fca758d46f4a45291b65e360c739c576742d6a12c2ebf2f70886a241424f75c33025aae6ff00245056a622d4f16df7df944a6bbdea8823c5d3640d82c3c939e1668962033863d51accfb6dd02b443b2078e68aa5e9e5b4fe226c2ab670713e214530a4c1c73989760cf8ba08a87c20f0a03e9fbc22c8be718d4af14b1a729d7494aa067bf9a9cb42e578bef9fea60e332c2a18e9de54188a083b997ae70b1f4a88f7d2f5e201c33e138b0b79e33c3c099873ec02acfa9da57712ea6115ee553bad7ca4ee775eeb1319c95a02c687060b0b47bd8d004e6b8f6b5a7095dd210c108d9800be471acac750ad33d332138e0fecac173dcc6b1b1aa4fd55d374f22db4f59fde21dfc2de77a2db12a8f1681180428483b2ac134078faf056ad511a515e0824d40dfd63f602d3dabe2be33c3bc5d8408519dbba5478195eb23095b79d7bb705bd0868899e0a12d06cc2d052f5c01c71937082662f6209697a5e6908aeafba6465897fae1b9fbbe42fadc52a823ce2aa191375ad2b93462c84fb74a9eb02b9391a34a3a8ad2c83d646bffa641e78245568fca9f35a07dad31baa7814de590ac63ed4182034bf4ff68be4b428ce1ea679ad146d378cf5de1049b5abe542684cb49183281d68596059691ded3e65913c84c63d49000a927bb6af9d3e2577ee992c9c5a0f952a84e3006a283fd02907421edd90bd5da21386b533a68b933914e0a7b7fa27535008310e0d40d1d6911573cd1d1900d085c509854c415c244aa3a9a937ca29d3f809ec12fc677c1fb70762c4e0e0c463702bdad82e2a6b6bcd2d83c7710a9013497c0a639e5f379e668eea4f4222f9f0f2d00a1ce438c8305d7b04cdb2380f50ee7d774149762d8f40061b743bf9dc7f8411f766e75e9b1c6fba94a1cae6171c27821fcf9b4b9bd3278066aa900f111cdd97cbffe9fad3aa7b5096457677cc544091727d6dfd738e9e2669288182620e3e0d161a0f2f58336f14def91d826be5623970860f0e847d894701e130ccbc822c1c550a4ad6a3be48e905f2fe8d1e837d246f767b0c8454827228c82103a612f405bf7f867ac69a28f880f843e26054012f33273b36870b9b6a82353457cdce1f49301051219", "openwall"},
```

Comment:
But you are sending a `0`. 

Fixing the `if` or the test string makes it work.

Reply:
after testing stuf I forgot to change it back, thx for your hint 😄